multimodal-search-tirg/README.MD

```diff
@@ -57,16 +57,23 @@ bash ./get_data.sh data/
 
 Alternatively, You can [Download](https://drive.google.com/drive/folders/0B4Eo9mft9jwoamlYWFZBSHFzV3c) and extract the data from google drive.
 
-## Build encoder images
-This will download the TIRG model + code and build the image
+## Build encoder images from Jina hub
```

Comment:
Do we actually still need to pull and run it explicitly?
I thought it is enough to specify `uses: docker://jinahub/pod.encoder.tirgimageencoder:0.0.4-0.9.17` in the `flow-index.yml`

Reply:
ah that's good to know, thanks